bin/validator_tests.py

```diff
@@ -1,24 +1,45 @@
+import validate
+
 from collections.abc import Callable, Sequence
 from typing import Final, Optional, TypeVar
+from validate import AnswerValidator, InputValidator, OutputValidator
```

Comment:
Can the `import validate` and `from validate import ...` be merged? :innocent:

Reply:
what exactly do you mean?